node/consensus/engine.go

```diff
@@ -313,6 +316,8 @@ func New(cfg *Config) *ConsensusEngine {
 	return ce
 }
 
+var initialHeight int64 = 0 // TODO: get it from genesis?
```

Comment:
Ah, good point.  Yeah we should add this to the genesis config struct.  Can do later though.